src/applications/claims-status/components/AppealEventItem.jsx

```diff
@@ -28,7 +29,7 @@ export default class AppealEventItem extends React.Component {
     });
 
     return (
-      <li className="event-item" onClick={this.toggle}>
+      <li className="event-item" onClick={this.toggle} onKeyPress={this.toggle}>
```

Comment:
This would indiscriminately allow _any_ keypress to toggle the list. I think the lint error is meant to not allow the `<li>` to be used as a button. Maybe we could try to add a button that fills the header and allows toggling?

Reply:
@Mottie In the description I noted that the component isn't actually used. If I was used, I think it's worth fixing. But I wasn't sure if it's worth going through the process of working with design on this if nobody would be able to view the component anyway.

Reply:
Ahh yes, I believe that is true. Good point LOL.